api/src/test/resources/io/kaoto/backend/api/resource/eip.kamelet.yaml

```diff
@@ -22,6 +22,22 @@ spec:
           constant: '3'
           copy: true
           steps:
+          - saga:
+              completion:
+                uri: direct:completion
+              propagation: MANDATORY
+              compensation:
+                uri: direct:compensation
+              completion-mode: MANUAL
+              steps:
+                - to:
+                    uri: direct:something
+              option:
+                - key: o1
+                  simple: ${body}
+                - key: o2
+                  expression:
+                    simple: ${body}
```

Comment:
what do you think to add also the `sage-service`, `timeout` and `description` attributes to cover all cases with defined values?

Reply:
Sure. I am just copying/pasting camel tests on this file. But we can add them.